plotter/scattercolor_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"gonum.org/v1/plot/palette"
```

Comment:
this `gonum.org/v1/plot/palette` import should be part of the `gonum.org` import block below.